### ISSUE TITLE
Allow EdgeText to render child elements

### ISSUE THREAD

**Issue by gfox1984** [wbkd/react-flow#1073]
Hello, me again!

Going one step further than in https://github.com/wbkd/react-flow/issues/1045, that would be great if the `EdgeText` component could render the specified children inside the wrapping &lt;g&gt; element (eg: just after the &lt;rect&gt; and &lt;text&gt;). Eg:

```
<EdgeText
  x={x}
  y={y}
  label="Hello world"
>
  <circle cx="40" cy="40" r="25" />
</EdgeText>
```

The reason I'm asking is that I'm going into some customisation that requires drawing some extra gimmicks next to the edge text. Since I'm already happily to use the `EdgeText` component, which handles the drawing of text and background rectangle perfectly, it would be ideal to keep using it instead of duplicating its code inside my own component.

Note: I'm happy to handle the positioning of the child elements myself

**Comment by moklick** [comment]
hey @gfox1984 :)

good idea, I will add it!

**Comment by gfox1984** [comment]
Actually, I'm thinking it would be helpful to raise an event on `EdgeText` when you update its state (`setEdgeTextBbox`), along with the rectangle you're using. It would save the hassle of recomputing it and getting the timing right when we want to re-position the children...

**Comment by moklick** [comment]
Released in 9.5.1. The children should re-render when we update the sizes.
